openfeature/provider/__init__.py

```diff
@@ -17,6 +21,20 @@ class ProviderStatus(Enum):
     STALE = "STALE"
     FATAL = "FATAL"
 
+    __events__: typing.ClassVar[typing.Dict[ProviderEvent, str]] = {
+        ProviderEvent.PROVIDER_READY: READY,
+        ProviderEvent.PROVIDER_ERROR: ERROR,
+        ProviderEvent.PROVIDER_ERROR: FATAL,
```

Comment:
this doesn't work, either we emit `ERROR` or `FATAL` for `PROVIDER_ERROR`

Reply:
good catch, did the change mechanically and forgot it wasn't a bijection